[2025-03-14T16:11:31.609+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [queued]>
[2025-03-14T16:11:31.621+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [queued]>
[2025-03-14T16:11:31.622+0000] {taskinstance.py:2171} INFO - Starting attempt 16 of 17
[2025-03-14T16:11:31.639+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): run_script> on 2025-03-10 22:00:00+00:00
[2025-03-14T16:11:31.648+0000] {standard_task_runner.py:60} INFO - Started process 884 to run task
[2025-03-14T16:11:31.652+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'run_script', 'scheduled__2025-03-10T22:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmpvbgp9t2m']
[2025-03-14T16:11:31.653+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask run_script
[2025-03-14T16:11:31.724+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [running]> on host 91a15d92d4ed
[2025-03-14T16:11:31.847+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='16' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T22:00:00+00:00'
[2025-03-14T16:11:31.848+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-14T16:11:31.849+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        python /opt/***/external_scripts/gen_transaction_data.py 2025-03-11\n        ']
[2025-03-14T16:11:31.863+0000] {subprocess.py:86} INFO - Output:
[2025-03-14T16:11:32.642+0000] {subprocess.py:93} INFO - Data retrieved! (20000, 6)
[2025-03-14T16:11:32.642+0000] {subprocess.py:93} INFO - Data saved! /tmp/***tmp5bc7n5jm
[2025-03-14T16:11:32.727+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-14T16:11:32.766+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=run_script, execution_date=20250310T220000, start_date=20250314T161131, end_date=20250314T161132
[2025-03-14T16:11:32.819+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-14T16:11:32.842+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
